crates/binjs_io/src/entropy/mod.rs

```diff
@@ -55,19 +55,20 @@
 //! and/or custom dictionary later.
 
 pub mod model;
+
 mod predict;
-mod tree;
-pub mod read;
-pub mod write;
+// mod tree;
```

Comment:
I wonder what happens to this file...

Reply:
It has become irrelevant. I have upstreamed all the important parts into the auto-generated ast.rs.